features/other_a/invite_use.feature

```diff
@@ -29,3 +29,17 @@ I want to use an invitation to create an account
     And I go to SOME_USER2's invitations page
   Then I should be on Scott's user page
     And I should see "Sorry, you don't have permission to access the page you were trying to reach."
+
+  Scenario: Viewing invitation details when the invitee has deleted their account
+    Given the user "creator" exists and is activated
+      And the user "invitee" exists and is activated
+      And an invitation created by "creator" and used by "invitee"
+      And I am logged in as "creator"
+    When I view the most recent invitation for "creator"
+    Then I should see "invitee"
+    When I am logged in as "invitee"
+      And "invitee" deletes their account
+      And I am logged in as "creator"
+      And I view the most recent invitation for "creator"
+    Then I should see "Deleted User"
+      But I should not see "invitee"
```

Comment:
I know it should be fine, but can we also check that "invitee" doesn't appear in the table on your manage invitations page (`/users/USERNAME/invitations/manage)`?